nilearn/tests/test_region.py

```diff
@@ -305,12 +306,14 @@ def test_signal_extraction_with_maps_and_labels():
     mask_data = (labels_data == 1) + (labels_data == 2) + (labels_data == 5)
     mask_img = nibabel.Nifti1Image(mask_data.astype(np.int8),
                                    labels_img.get_affine())
-    labels_signals, labels_labels =\
-                    region.img_to_signals_labels(fmri_img, labels_img,
-                                                 mask_img=mask_img)
+    # Ignore a known warning about divide-by-zero on our dummy data.
+    labels_signals, labels_labels = \
+        assert_warns_regex(RuntimeWarning, '',
```

Comment:
Is there no meaningful string you could use here for the warning message? In case there is none, you could consider using assert_warns instead.